### ISSUE TITLE
Incorrect note block sound with mob head on top

### ISSUE THREAD

**Issue by XEnderB0yX** [PluginBugs/Issues-ItemsAdder#3584]
### Terms

- [X] I'm using the very latest version of ItemsAdder and its dependencies.
- [X] I am sure this is a bug and it is not caused by a misconfiguration or by another plugin.
- [X] I already searched on this [Github page](https://github.com/PluginBugs/Issues-ItemsAdder/issues) to check if the same issue was already reported.
- [X] I already searched on the [plugin wiki](https://itemsadder.devs.beer/) to know if a solution is already known.
- [X] I already searched on the [forums](https://forum.devs.beer/) to check if anyone already has a solution for this.

### What happened?

If the mob's head is on a note block, then the note block will start making other sounds, depending on how many times the player clicked on it. The first time you press it, the sound will slow down, and each time you press it again, the sound will become faster

### Steps to reproduce the issue

1) Put the mob's head on the note block
2) Start clicking on it several times (you will notice how the sound changes with each click)

### ItemsAdder Version

ItemsAdder version 3.6.3-beta-14

### ProtocolLib Version

ProtocolLib version 5.2.0-SNAPSHOT-675

### LoneLibs Version

LoneLibs version 1.0.45

**Comment by Andre601** [comment]
Could you share a video of this behaviour? I have a hard time understanding what the issue exactly is.

**Comment by XEnderB0yX** [comment]
Uploading output(compress-video-online.com) (1) (1).mp4…

**Comment by XEnderB0yX** [comment]
https://github.com/PluginBugs/Issues-ItemsAdder/assets/144527158/46a3495c-de56-40bb-beaa-1f989715f0ec

**Comment by Andre601** [comment]
I don't think this is a bug? Pretty sure this is normal vanilla behaviour. To make sure, check if the same happens in singleplayer.

**Comment by XEnderB0yX** [comment]
This does not happen in a single player game

**Comment by Andre601** [comment]
Alright, thanks for the clarification. LoneDev may have some input to give once he has time.
